### PR TITLE
Feature/delay

### PR DESCRIPTION
**Pull request by jamonholmgren**
Adds `app.after(/delay) and app.every`.

```ruby
rmq.app.after 3 do
  puts "Time waits for no one"
end

rmq.app.every 0.1 do
  puts "Jamon is cool"
end
```

### CODE REVIEW

**Comment by GantMan** [comment]
LOVE IT, but tests failed.

**Comment by jamonholmgren** [comment]
screen shot 2015-03-05 at 5 01 41 pm

**Comment by GantMan** [comment]
@jamonholmgren it looks like the comparison of 0.011084.<(0.011)  is causing some kind of error on this machine...   Maybe a little more leeway?

File: spec/app.rb [hunk 1]
```diff
@@ -44,6 +44,31 @@
...
+    wait {}
+  end
+
+  it 'should fire twice 0.01 seconds apart' do
+    before_time = Time.now
+    count = 0
+    timer = @app.every 0.01 do
+      after_time = Time.now
+      (after_time - before_time).should.be > 0.009
+      (after_time - before_time).should.be < 0.011
```
Comment (markrickert) [inline spec/app.rb]: `.should.be.close 0.011, 0.01`

**Comment by GantMan** [comment]
Does this issue need some assistance?

**Comment by twerth** [comment]
Gant would you check this and merge it if it passes

**Comment by GantMan** [comment]
Yeah, I'll try to investigate the test.  Even if it's a great feature we might need to mute the test a bit if it causes master to fail tests.   I'll check into it.

**Comment by jamonholmgren** [comment]
@GantMan I just totally ignored you after you told me what was going on. Fixed now.

**Comment by GantMan** [comment]
@jamonholmgren that's ok, I think that means we're in a relationship now.

**Comment by jamonholmgren** [comment]
crying

**Comment by GantMan** [comment]
wait... was this ever fixed?   I'm unable to run tests locally now, and this is crashing it.

**Comment by GantMan** [comment]
Turns out it SOMETIMES fails :/

Might be worth adding another level of fuzziness.   .009 instead of .01

**Comment by jamonholmgren** [comment]
@GantMan Yours is failing with 0.009 seconds? It should never go below 0.01... O_o

**Comment by GantMan** [comment]
I made a PR, and I linked to travis failing on 0.008!   So I moved the test to 0.007.    I know it sucks, but travis is crapping out on all my PRs today, bc of this.

**Pull request merged**